FirebaseAI.podspec

```diff
@@ -43,10 +43,10 @@ Build AI-powered apps and features with the Gemini API using the Firebase AI SDK
   s.tvos.framework = 'UIKit'
   s.watchos.framework = 'WatchKit'
 
-  s.dependency 'FirebaseAppCheckInterop', '~> 12.0'
-  s.dependency 'FirebaseAuthInterop', '~> 12.0'
-  s.dependency 'FirebaseCore', '~> 12.0.0'
-  s.dependency 'FirebaseCoreExtension', '~> 12.0.0'
+  s.dependency 'FirebaseAppCheckInterop', '12.0.0'
+  s.dependency 'FirebaseAuthInterop', '12.0.0'
+  s.dependency 'FirebaseCore', '12.0.0'
```

Comment:
The one downside is that a patch release would require re-releasing all pods, since at minimum FirebaseCore will need to change in the patch release. 

As a result, I think I'd like to pin to the point release for internal dependencies. Patches are rare, so it will function much like SPM most of the time, but offer a little more flexibility for when we do need to make patch releases.

I figure patch release change should be reviewed closely so the possibility of introducing a bug at the patch level is low (and instead more common and risky to introduce at the minor level where our internal deps are currently pinned).

Reply:
By "point release", do you mean patch version or minor version?

The old code pins to a minor version. The new code pins to a patch version.

Reply:
Yeah, my bad, I meant patch version, not point. Old code pins to a minor version. New code pins to a patch version. My thinking now is that new code should **float** to a patch version.

Reply:
Agreed. So you'll change it back to `'~> 12.0.0'`?

Reply:
Yes. Changing it back to `'~> 12.0.0'` and making that the standard for other internal dependencies besides `FirebaseCore`.